docs/source/details/backendconfig.rst

```diff
@@ -287,3 +287,76 @@ Explanation of the single keys:
   In "template" mode, only the dataset metadata (type, extent and attributes) are stored and no chunks can be written or read (i.e. write/read operations will be skipped).
 * ``json.attribute.mode`` / ``toml.attribute.mode``: One of ``"long"`` (default in openPMD 1.*) or ``"short"`` (default in openPMD 2.* and generally in TOML).
   The long format explicitly encodes the attribute type in the dataset on disk, the short format only writes the actual attribute as a JSON/TOML value, requiring readers to recover the type.
+
+Dataset-specific configuration
+------------------------------
+
+Some configuration options should be applicable on a per-dataset basis.
+Most dataset-specific configuration options supported by the openPMD-api are additionally backend-specific, being format-specific serialization instructions such as compression or chunking.
+Such dataset-specific and backend-specific configuration is hence specified under the key path ``<backend>.dataset``, e.g.:
```

Comment:
The initial examples are not really showing dataset specific (filtered/selected) examples, so I reworded this a little to avoid confusion:
```suggestion

All dataset-specific and backend-specific configuration is specified under the key path ``<backend>.dataset``.
Without filtering by dataset name (see the ``select``` key below) this looks like:
```